src/lib/onboarding/router.ts

```diff
@@ -97,6 +103,31 @@ router.post('/submit-verified', async (req, res) => {
   }
 });
 
+/// FALLBACK ONLY
+router.post('/submit-with-sig', async (req, res) => {
```

Comment:
You can type all these `router.get|post<>` if you want
Not necessary, of course

Reply:
yap missing everywhere, will add on a shiny day ;)